cmswww/cmd/cmswwwdataload/client/client.go

```diff
@@ -119,24 +120,19 @@ func (c *Client) ExecuteCliCommand(reply interface{}, verify verifyReply, args .
 		return err
 	}
 
-	var stderrBytes, stdoutBytes []byte
-	var errStderr, errStdout error
-	go func() {
-		stderrBytes, errStderr = ioutil.ReadAll(stderr)
-		if errStderr == nil && len(stderrBytes) > 0 {
-			errStderr = fmt.Errorf("unexpected error output from %v: %v", cli,
-				string(stderrBytes))
-		}
-	}()
+	stderrBytes, errStderr := ioutil.ReadAll(stderr)
```

Comment:
I think this change may be causing an issue with the dataload. If you run it with tests, like this:

```
cmswwwdataload -v -includetests
```

it gets stuck (I think) when it calls `testInventory`. This is the last of the output I see, and the program doesn't seem to terminate:

```
2019/02/07 23:00:46 Fetching all invoices
2019/02/07 23:00:46   $ cmswwwcli --host https://127.0.0.1:4443 --jsonout invoices
```

Reply:
Strange.  I've tried this again a handful of times using ```cmswwwdataload --verbose --deletedata --includetests```, but haven't seen it get stuck.

You're trying this on mac, right?  I might have to see if I can get my hands on one to test with.  Just to eliminate a couple of things:

- Did you update the vendor folder with ```GO111MODULE=on go mod vendor```?
- What's your Go version?

Reply:
Actually, I'm not even seeing that line in my output ('Fetching all invoices').  When I try to run it manually though, it works fine.  Do you know why I might not be seeing that particular test?

Reply:
@sndurkin Are you able to provide me with a little more information to help me track down what's going wrong here please?  Have some questions above for you in this conversation thread.

Reply:
Hey @orthomind, sorry for the lack of response, been really busy lately.

> You're trying this on mac, right?

I'm on Windows.

> Did you update the vendor folder with `GO111MODULE=on go mod vendor`

Yeah, I did `GO111MODULE=on go mod vendor && go install ./...`.

> What's your Go version?

`go version go1.11.4 windows/amd64`

> Actually, I'm not even seeing that line in my output ('Fetching all invoices').

I'm not really sure why you're not seeing that output. If you look at `cmswwwdataload/main.go`, you can see that if `cfg.IncludeTests` is true, it calls `testInventory`, which calls `GetAllInvoices`, which prints the 'Fetching all invoices' log.

Here's the full output that I'm seeing: https://gist.github.com/sndurkin/86b71cf5787b0fad988d1169a4b304f0